brainscore_vision/models/temporal_model_openstl/model.py

```diff
@@ -105,23 +105,6 @@ def process_output(layer, layer_name, inputs, output):
         kwargs = {}
         weight_name = "kitticaltech_predrnn_one_ep100.pth"
 
-    elif identifier == "PredNet":
-        layer_activation_format = {
-            **{f"layer{i}": "TCHW" for i in range(4)},
-            "layer5": "TCHW"
-        }
-
-        def process_output(layer, layer_name, inputs, output):
-            if layer_name.startswith("cell_list"):
-                h, c = output
-                return c
-            else:
-                return output
-        
-        wrapper_cls = LSTMWrapper
-        kwargs = {}
-        weight_name = "kitticaltech_prednet_one_ep100.pth"
```

Comment:
maybe check with the plugin author to make sure we're okay to delete this and why it's registry is commented out

Reply:
@YingtianDt I removed the above code due to `# model_registry["PredNet"] = lambda: commit_model("PredNet")` being commented out in the init file. How would you like to proceed? Should `PredNet` be commented out?

Reply:
Yes, please remove this part since "PredNet" is not supported by openstl.